apps/webapp/app/routes/api.v1.integrations.$integrationSlug.connections.$connectionId.ts

```diff
@@ -0,0 +1,95 @@
+import { LoaderArgs, json } from "@remix-run/server-runtime";
+import { z } from "zod";
+import { prisma } from "~/db.server";
+import { resolveApiConnection } from "~/models/runConnection.server";
+import { authenticateApiRequest } from "~/services/apiAuth.server";
+import { apiCors } from "~/utils/apiCors";
+
+const ParamsSchema = z.object({
+  integrationSlug: z.string(),
+  connectionId: z.string(),
+});
+
+export async function loader({ request, params }: LoaderArgs) {
+  const authenticationResult = await authenticateApiRequest(request);
+  if (!authenticationResult) {
+    return apiCors(
```

Comment:
We should only use the `apiCors` function if we want to allow browsers to access the endpoint. It doesn't seem like we do in this situation?